Documentation/Books/Manual/Uninstallation/README.md

```diff
@@ -0,0 +1,31 @@
+Uninstallation
+==============
+
+Uninstallation depends on the method used to install ArangoDB, and on the
+operating system in use, and typically consists of the following high-level steps:
+
+**If an installation package is used:**
+
+- Uninstallation of the ArangoDB package, which may or may not remove the
+  database directory and the configuration files in addition to the ArangoDB
+  _binaries_, depending on how the uninstallation process is started and on
+  the operating system in use.
```

Comment:
This could frighten users. I think we should be more explicit, and they which packages ask you whether you want to remove your data as well (all?).

Reply:
Created ticket: https://github.com/arangodb/planning/issues/3317